test/Microsoft.AspNet.Mvc.Localization.Test/DependencyInjection/MvcLocalizationServiceCollectionExtensionsTest.cs

```diff
@@ -116,11 +116,13 @@ public void AddCustomLocalizers_AfterMvcLocalization_AddsNeededServices()
 
             Assert.Equal(typeof(IConfigureOptions<RazorViewEngineOptions>), services[0].ServiceType);
             Assert.Equal(ServiceLifetime.Singleton, services[0].Lifetime);
-            Assert.Equal(0, ((IConfigureOptions<RazorViewEngineOptions>)services[0].ImplementationInstance).Order);
+            // REVIEW: What should this be replaced with?
+            //Assert.Equal(0, ((IConfigureOptions<RazorViewEngineOptions>)services[0].ImplementationInstance).Order);
 
             Assert.Equal(typeof(IConfigureOptions<RazorViewEngineOptions>), services[1].ServiceType);
             Assert.Equal(ServiceLifetime.Singleton, services[1].Lifetime);
-            Assert.Equal(-1000, ((IConfigureOptions<RazorViewEngineOptions>)services[1].ImplementationInstance).Order);
+            // REVIEW: What should this be replaced with?
+            //Assert.Equal(-1000, ((IConfigureOptions<RazorViewEngineOptions>)services[1].ImplementationInstance).Order);
```

Comment:
Any way to confirm the previously-earlier (`-1000`) service implementation is now at `services[0]`?